docs/source/tips-and-tricks.rst

```diff
@@ -4,10 +4,77 @@ Tips and Tricks
 .. py:currentmodule:: htmap
 
 
+.. _cli-tips:
+
+Separate job submission/monitoring/collection
+---------------------------------------------
```

Comment:
I'm not sure this section is accurate and that it's actually a tip/trick. Can you review this section?